<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_form</class>
 <widget class="QWidget" name="main_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="top_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_musteri">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Müşteri</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cmb_musteri"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lbl_sozlesme">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Sözleşme</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="cmb_sozlesme">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="lbl_service_type">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string>Hizmet Tipi</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="cmb_service_type">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>28</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Seçiniz..</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ÖĞRENCİ TAŞIMA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PERSONEL TAŞIMA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ARAÇ KİRALAMA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DİĞER</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QFrame" name="middle_s_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="txt_kalem_sec">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>425</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>425</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>İŞ KALEMİ (HAT) VEYA DURAK NOKTASI YAZ..</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListView" name="list_kalemler">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>425</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>425</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="movement">
         <enum>QListView::Movement::Static</enum>
        </property>
        <property name="layoutMode">
         <enum>QListView::LayoutMode::Batched</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="middle_n_frame">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTableWidget" name="table_sefer">
        <property name="inputMethodHints">
         <set>Qt::InputMethodHint::ImhNone</set>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <column>
         <property name="text">
          <string>ROTA</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>GİRİŞ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ÇIKIŞ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ARAÇ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>KPST.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>SÜRÜCÜ</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QFrame" name="bottom_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>35</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_kaydet">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>KAYDET</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_satir_sil">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>SATIR SİL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_delete_all">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>TÜM SEFERLERİ SİL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>186</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
